<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceivePaymentDialog</class>
 <widget class="QDialog" name="ReceivePaymentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Receive Payment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Payment For:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLineEdit" name="txtYear">
       <property name="text">
        <string>2014</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Year:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Month:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Unit:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Date Received:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tenant Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Receipt No:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Payment Description:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QComboBox" name="cboMonth">
       <item>
        <property name="text">
         <string>January</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>February</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>March</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>April</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>May</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>June</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>July</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>August</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>September</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>October</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>November</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>December</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QDateEdit" name="dtpDate">
       <property name="displayFormat">
        <string>dd-MMM-yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="spinAmount">
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="5">
      <widget class="QPlainTextEdit" name="txtDescription"/>
     </item>
     <item row="2" column="1" colspan="5">
      <widget class="QLineEdit" name="txtReceiptNo"/>
     </item>
     <item row="1" column="1" colspan="5">
      <widget class="QComboBox" name="cboUnit"/>
     </item>
     <item row="0" column="1" colspan="5">
      <widget class="QComboBox" name="cboTenant"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Payment Mode:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="cboPaymentMode">
       <item>
        <property name="text">
         <string>Cash</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cheque</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mobile Payment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Credit Card</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Electronic Transfer</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Cheque No.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="3">
      <widget class="QLineEdit" name="txtChequeNo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cboTenant</tabstop>
  <tabstop>cboUnit</tabstop>
  <tabstop>txtReceiptNo</tabstop>
  <tabstop>dtpDate</tabstop>
  <tabstop>cboMonth</tabstop>
  <tabstop>txtYear</tabstop>
  <tabstop>spinAmount</tabstop>
  <tabstop>cboPaymentMode</tabstop>
  <tabstop>txtChequeNo</tabstop>
  <tabstop>txtDescription</tabstop>
  <tabstop>cmdSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
